src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+2.0.21
+++++++
+vmss: ensure app-gateway has a name when defaults to it for large scalesets
 
 2.0.19
```

Comment:
Seems like we're missing a 2.0.20 here.  It can just say "Minor fixes."

Reply:
2.0.20 is released today from the dev branch